runtime/j9vm/javanextvmi.cpp

```diff
@@ -751,4 +751,13 @@ JVM_VirtualThreadDisableSuspend(
 }
 #endif /* JAVA_SPEC_VERSION >= 22 */
 
+#if JAVA_SPEC_VERSION >= 23
+JNIEXPORT jint JNICALL
+JVM_GetCDSConfigStatus()
+{
+	/* OpenJ9 does not support CDS, so we return false unconditionally. */
+	return JNI_FALSE;
```

Comment:
The return value is not a boolean, but a bit-mask; see `jdk.internal.misc.CDS.IS_xxx`.

Reply:
Fixed, return `0` instead.